exampleSite/content/sdgs/_index.en.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Sustainable Development Goals"
+date: 2018-12-29T11:02:05+06:00
+icon: "ti-link"
+description: "Lorem ipsum dolor sit amet ipsum dolor sit amet ipsum dolor sit amet"
+type : "docs"
```

Comment:
This is invoking the documentation page layout. I'm not sure the redirect will work if a user ends up on the `/sdgs/` page. Does this need to be a new layout instead of an archetype?